frontend/resources/translations/de-DE.json

```diff
@@ -637,7 +827,231 @@
+    "dateTimeFormatTooltip": "Ändere das Datums- und Uhrzeitformat in den Einstellungen"
+  },
+  "relativeDates": {
+    "yesterday": "Gestern",
+    "today": "Heute",
+    "tomorrow": "Morgen",
+    "oneWeek": "1 Woche"
+  },
+  "notificationHub": {
+    "title": "Benachrichtigungen",
+    "emptyTitle": "Alles erledigt!",
+    "emptyBody": "Keine wartenden Benachrichtigungen oder Aktivitäten. Erfreue dich an der Ruhe.",
+    "tabs": {
+      "inbox": "Eingang",
+      "upcoming": "Anstehend"
+    },
+    "actions": {
+      "markAllRead": "Markiere alle als gelesen",
+      "showAll": "Alle",
+      "showUnreads": "Ungelesen"
+    },
+    "filters": {
+      "ascending": "Aufsteigend",
+      "descending": "Absteigend",
+      "groupByDate": "Guppiere nach Datum",
+      "showUnreadsOnly": "Zeige nur ungelesene",
+      "resetToDefault": "Auf Standard zurücksetzen"
+    }
+  },
+  "reminderNotification": {
+    "title": "Erinnerung",
+    "message": "Denke daran, dies zu prüfen, bevor du es vergisst!",
+    "tooltipDelete": "Löschen",
+    "tooltipMarkRead": "Als gelesen markieren",
+    "tooltipMarkUnread": "Als ungelesen markieren"
+  },
+  "findAndReplace": {
+    "find": "Suchen",
+    "previousMatch": "Vorheriges Vorkommen",
+    "nextMatch": "Nächstes Vorkommen",
+    "close": "Schließen",
+    "replace": "Ersetzen",
+    "replaceAll": "Alles ersetzen",
+    "noResult": "Keine Ergebnisse",
+    "caseSensitive": "Groß-/Kleinschreibung beachten"
+  },
+  "error": {
+    "weAreSorry": "Es tut uns leid",
+    "loadingViewError": "Wir haben Probleme, diese Ansicht zu laden. Bitte prüfe deine Internetverbindung, aktualisiere die Anwendung und zögere nicht, das Team zu kontaktieren, sollte der Fehler weiterhin auftreten."
+  },
+  "editor": {
+    "bold": "Fett",
+    "bulletedList": "Auflistung",
+    "checkbox": "Checkbox",
+    "embedCode": "Eingebetteter Code",
+    "heading1": "H1",
+    "heading2": "H2",
+    "heading3": "H3",
+    "highlight": "Markieren",
+    "color": "Farbe",
+    "image": "Bild",
+    "italic": "Kursiv",
+    "link": "Verknüpfung",
+    "numberedList": "Aufzählung",
+    "quote": "Zitat",
+    "strikethrough": "Durchgestrichen",
+    "text": "Text",
+    "underline": "Unterstrichen",
+    "fontColorDefault": "Standard",
+    "fontColorGray": "Grau",
+    "fontColorBrown": "Braun",
+    "fontColorOrange": "Orange",
+    "fontColorYellow": "Gelb",
+    "fontColorGreen": "Grün",
+    "fontColorBlue": "Blau",
+    "fontColorPurple": "Lila",
+    "fontColorPink": "Rosa",
+    "fontColorRed": "Rot",
+    "backgroundColorDefault": "Standard Hintergrund",
+    "backgroundColorGray": "Grauer Hintergrund",
+    "backgroundColorBrown": "Brauner Hintergrund",
+    "backgroundColorOrange": "Orangener Hintergrund",
+    "backgroundColorYellow": "Gelber Hintergund",
+    "backgroundColorGreen": "Grüner Hintergrund",
+    "backgroundColorBlue": "Blauer Hintergrund",
+    "backgroundColorPurple": "Lilaner Hintergrund",
```

Comment:
"lila"
Weil es mich interessiert hat, habe ich recherchiert (https://de.wiktionary.org/wiki/Flexion:lila):
In der Standardsprache ist das Adjektiv rosa/lila indeklinabel (ein rosa/lila Pullover). In der Umgangssprache existieren dagegen flektierte Formen (rosaner/lilaner).
Das bitte nur als Hinweis und nicht als "Klugscheißerei" verstehen. Das ist in diesem Zusammenhang sicherlich Geschmackssache ;-)